docs/guidance/rayservice.md

```diff
@@ -0,0 +1,158 @@
+## Ray Service (alpha)
+
+> Note: This is the alpha version of Ray Service. There will be ongoing improvements for Ray Service in the future releases.
```

Comment:
`Service -> Services`, almost everywhere, including in all three spots above this comment.
https://github.com/ray-project/kuberay/pull/285#issuecomment-1145529800 :p